src/networking/payloads/connectiontype.rs

```diff
@@ -0,0 +1,25 @@
+#[derive(Debug, PartialEq)]
+pub enum ConnectionType {
+  UNKNOWN,
+  PUBLIC,
+  SYMMETRICNAT,
+}
+
+impl ConnectionType {
+  pub fn encode(&self) -> (bool, bool) {
+    match self {
+      ConnectionType::UNKNOWN => (false, false),
```

Comment:
Those booleans look like magic numbers right now, maybe you could document them?

Reply:
will do